include/hdf5_particle.h

```diff
@@ -39,8 +38,33 @@ typedef struct HDF5Particle {
   mpm::Index cell_id;
   // Status
   bool status;
+  // Material id
+  unsigned material_id;
+  // Number of state variables
+  unsigned nstate_vars;
+  // State variables (init to zero)
+  double svars[20] = {0};
```

Comment:
Will the state variables equal to 0, i.e. uninitialized, be printed too in the write_particle_hdf5?